### DISCUSSION TITLE
Scaling UI keybind conflict

### DISCUSSION THREAD

**Discussion by TobiasTagarsi** [ppy/osu#19873] [Q&A] [answered]
Hello,

I would like to use "ctrl -" and "ctrl +" as my osu! left and right button keybinds. However, regardless of being bound or not, both keybinds will always scale UI and I cannot find a way to unbind them after having a look through forum posts. I have searched through osu!lazer configuration files, but don't know if any of the settings would resolve my issue.

Thanks for you time.

**Comment by peppy** [comment, marked as answer]
We can probably add those as customisable bindings so you can unbind them, but for my own sanity can you explain why you'd want this?

**Reply by TobiasTagarsi** [reply]
Thank you for your response, I appreciate your time!
Below is the explaination I posted to the osu forum (where I mistakenly posted this problem before here):

I have a weird situation where I am using two xp-pen StarG640 graphics tablets. One has OpenTabletDriver active and the other does not (only one tablet is seen at a time. I've tried to get round this with the configuration tool but both tablets have the same IDs and are both from the same factory). The one with the driver active is my pen tablet, and the one without is my button tablet (so I cannot change the hardware buttons for my button tablet). The default button configurations that I want to use are "ctrl -" and "ctrl +". At the moment I'm getting by using the middle buttons which happen to be bound to "e" and "b", which is very uncomfortable to play osu with.

**Reply by peppy** [reply]
That's a bit unfortunate.

We've made these keys "system" level, which means it's not going to be easy to resolve this, unless we allow local overrides to take priority over system bindings. Depending on how often this comes up it may be a direction we can go in.

**Reply by TobiasTagarsi** [reply]
Ah, well, thank you for looking into my issue. I'll try and find an alternative solution.

**Reply by peppy** [reply]
You could use Microsoft Powertoys to remap the keys at an OS level, if that's an option.

**Reply by TobiasTagarsi** [reply]
After playing around with Powertools for a bit I found a solution that can fix shortcut/bind conflict issues:

Using peppy's suggestion, Powertools' keyboard manager, you can disable keys as well as remap them. In this case disabling left-ctrl fixed my issue, as + and - are bindable with no conflicts.
